app/controllers/options_controller.rb

```diff
@@ -52,12 +35,12 @@ def set_option
   end
 
   def falsify_all_others
-    falsify_others = Option.where("question_id = ? AND id NOT IN (?)" , params[:question_id], @option.id).update_all(:correct_key => false)
-    @option.correct_key = true if params[:correct_key]
+    falsify_others = Option.where("question_id = ? AND id NOT IN (?)" , params[:question_id], @option.id).update_all(:boolean_correct_answer => false)
+    @option.boolean_correct_answer = true if params[:boolean_correct_answer]
```

Comment:
instead of naming it as boolean_correct_answer, can we name it as is_correct_answer, that is a common practice with boolean db fields